foundry/applications/formApplications/imagePopout.d.ts

```diff
@@ -1,10 +1,96 @@
 /**
- * An Image Popout Application
- * Provides optional support to edit the image path being viewed
- * @param image - The image being viewed
- * @param options - Standard Application rendering options
- * @param onUpdate - An optional callback function which should be triggered if the Image path is edited
+ * An Image Popout Application which features a single image in a lightbox style frame.
+ * This popout can also be used as a form, allowing the user to edit an image which is being used.
+ * Furthermore, this application allows for sharing the display of an image with other connected players.
+ *
+ * @example
+ * ```javascript
```

Comment:
The code snippet should work just as well in Typescript, so we can just as well mark it as typescript.
```suggestion
 * ```typescript
```